sql/2019/10_SEO/10_05.sql

```diff
@@ -0,0 +1,13 @@
+#standardSQL
+
+#ld+json, microformatting, schema.org + what @type
+
+SELECT
+    url,
+    flattened_105
+FROM
+    `httparchive.pages.2019_07_01_desktop`
+CROSS JOIN
+    UNNEST(CAST(JSON_EXTRACT_SCALAR(REPLACE(JSON_EXTRACT_SCALAR(payload, '$._almanac'), '\\"','"'), "$['10.5']") as ARRAY)) as flattened_105
```

Comment:
I use REPLACE([escaped_json_string]) to remove the escaped "-character. Is there a better way to do this? It works, but I have the impression, it is not the best way to sanitise the JSON string.